oas

```diff
@@ -1,506 +1,354 @@
+                        "example": 25.3
+                    }
+                },
+                "description": "Data Model for GPS"
+            },
+            "Measurements": {
+                "title": "Measurements",
+                "type": "object",
+                "properties": {
+                    "temperature": {
+                        "type": "number",
+                        "description": "Measured temperature of the asset's temperature sensor",
+                        "example": 29.5
+                    },
+                    "humidity": {
+                        "type": "number",
+                        "description": "Measured humidity of the asset's humidity sensor",
+                        "example": 50.1
+                    },
+                    "pressure": {
+                        "type": "number",
+                        "description": "Measured pressure of the asset's pressure sensor",
+                        "example": 1020
+                    },
+                    "gps": {
+                        "$ref": "#/components/schemas/GPS"
+                    }
+                },
+                "description": "Data Model defining measurements for sensor data"
+            },
+            "SensorData": {
+                "title": "SensorData",
+                "type": "object",
+                "properties": {
+                    "timestamp": {
+                        "type": "string",
+                        "description": "Timestamp of the data point",
+                        "format": "date-time",
+                        "example": "2022-10-11T12:47:05.891905"
+                    },
+                    "measurements": {
+                        "$ref": "#/components/schemas/Measurements"
+                    }
+                },
+                "description": "Data Model for asset's sensor information"
             },
-            "example": [
-              "optional",
-              "additional",
-              "details"
-            ]
-          }
+            "TrackDataRequest": {
+                "title": "TrackDataRequest",
+                "required": [
+                    "GSIN",
+                    "SSCC"
+                ],
+                "type": "object",
+                "properties": {
+                    "GSIN": {
+                        "type": "string",
+                        "description": "GSIN of the returnable device. Required*",
+                        "example": "40212345678945787993"
+                    },
+                    "SSCC": {
+                        "type": "string",
+                        "description": "SSCC of the returnable asset. Required*",
+                        "example": "00512345678916851849"
+                    },
+                    "from": {
+                        "type": "string",
+                        "description": "The timestamp to start from. Optional",
```

Comment:
Don't use "Optional" in text. Make use of OAS "required" attribute instead
Explain what the default value is when not specified